ros/src/computing/planning/motion/packages/waypoint_follower/include/mpc_follower/mpc_utils.h

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2018-2019 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <cmath>
+#include <vector>
+#include <ros/ros.h>
+#include <eigen3/Eigen/Core>
+#include "autoware_msgs/Lane.h"
+#include <autoware_msgs/VehicleStatus.h>
+#include <tf2/utils.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <std_msgs/Float64MultiArray.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <geometry_msgs/TwistStamped.h>
+#include <amathutils_lib/amathutils.hpp>
+
+#include "mpc_follower/mpc_trajectory.h"
+
+namespace MPCUtils
+{
+
+/**
+ * @brief convert eular angle vector including +-2pi to 0 jump to continuous series data 
+ * @param a input angle vector
+ */
+void convertEulerAngleToMonotonic(std::vector<double> &a);
+
+
+/**
+ * @brief interpolate value vector at reference index
+ * @param index index vector related to object value (Eigen::Vector or std::vector)
```

Comment:
Please add `in` `out` `inout` to the documentation

Reply:
Done. Thanks!